infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -318,8 +339,21 @@ func (b *TFBlueprintTest) DefaultInit(assert *assert.Assertions) {
 	}))
 }
 
+// DefaultTerraformVetVet runs TF plan, TF show, and gcloud terraform vet on a blueprint.
+func (b *TFBlueprintTest) TerraformVet(assert *assert.Assertions) {
```

Comment:
Since we are in the tft package, I am leaning towards not having the `Terraform` prefix.

```suggestion
func (b *TFBlueprintTest) Vet(assert *assert.Assertions) {
```

Reply:
fixed